packages/theming/theme-types/src/Color.types.ts

```diff
@@ -522,6 +522,14 @@ export interface AliasColorTokens {
   /** @platform android, iOS */
   neutralForegroundOnColor?: ColorValue;
 
+  // TODO #2440: Add to android
+  /** @platform iOS */
+  neutralForegroundDarkStatic?: ColorValue;
+
+  // TODO #2440: Add to android
+  /** @platform iOS */
+  neutralForegroundLightStatic?: ColorValue;
```

Comment:
do we actually know for sure android needs it?

Reply:
There typically is a lag from iOS - Android figma on the tokens and values owing to design team functioning, but in almost all cases the tokens remain same for both mobile endpoints. So I would take these as valid changes for android.

Reply:
I would make the change for Android myself but the android token values in the design-tokens repo needs to be updated first, and there are tokens there that should be removed as well as added. I think it would make more sense to update Android altogether since there are updated controls that could be impacted by these token updates, which is why I made a github issue. @ankraj12 if there's a place in ADO that would make sense for me to create a work item, let me know.

Reply:
Sure Lynn, please feel free to add in a tracking item on me for this and android team an re-base the tokens we are using across deigns tokens repo. and FURN.